tools/tidy-manifests

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+tidy=/opt/ooce/bin/tidy
+
+ELEMENTS=(
+        '<service '
+        '<create_default_instance'
+        '<single_instance'
+        '<instance'
+        '<dependency'
+        '<dependent'
+        '<exec_method'
+        '<property_group'
+        '<template'
+        '<stability'
+        '<\/service'
+        '<\/instance'
+)
+
+echo "PROCESSING: $1"
+
+# tidy manifest according to doc/tidy.conf
+$tidy -q -config ./doc/tidy.conf -m $1
```

Comment:
Could you add something to this script so that it doesn't need to be run from the top-level of the checkout?
Often if I'm working on a package I'll be in `build/xxx` and want to be able to do `../../tools/tidy-manifest files/xxx.xml`

Something like
```
__SCRIPTDIR="${0%/*}"
```
then
```
$__SCRIPTDIR/../doc/tidy.conf
```
will work for ksh (if you change the shebang)